src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services1-2.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
```

Comment:
I am not sure. It's just may be, that docblocks look more structured to me than simple comments. I can change it to simple comment. Just docblock looks nicer to me ;)